### PR TITLE
The temperature gun no longer self charges. 

### PR DESCRIPTION
**Pull request by 1080pCat**
## What Does This PR Do

Removes the self charge ability of the temp gun, as well as fixes some strangeness with the temp changing, changes a comment to be actually correct.
Also changes the emag chat message from a 'caution'(normal white text) to a 'warning'(red italic text)
## Why It's Good For The Game

The temp gun is very powerful at what it does, now add onto the fact you don't need to worry about charge? you've got rather powerful weapon on your hands.
This pr makes it so they're not self sustaining. 

## Testing

Shot a bunch of skrells, and also made sure it was charging correctly, and changing temp correctly.
## Changelog
:cl:
tweak: The temp gun can no longer self charge.
tweak: Changes how the temp gun handles changing its temperature.
tweak: The chat message you get for emaging the temp gun looks scary, rather than just white text.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/energy/special_eguns.dm [hunk 1]
```diff
@@ -545,16 +545,15 @@
...
 
 	// Measured in Kelvin
 	var/temperature = T20C
 	var/target_temperature = T20C
 	var/min_temp = 0
 	var/max_temp = 500
 
-	/// How fast the gun recharges
-	var/recharge_multiplier = 1
+	// This var is actually how fast the gun changes temperature, uses this var for some god forsaken reason..
```
Comment (Henri215) [inline code/modules/projectiles/guns/energy/special_eguns.dm:555]:
```suggestion
	/// How fast the gun changes temperature
```
Not better like this?

File: code/modules/projectiles/guns/energy/special_eguns.dm [hunk 2]
```diff
@@ -597,17 +596,17 @@
...
 		max_temp = 1000
-		recharge_multiplier = 5  //so emagged temp guns adjust their temperature much more quickly
+		recharge_multiplier *= 5  //so emagged temp guns adjust their temperature much more quickly
 
 /obj/item/gun/energy/temperature/process()
 	..()
 	if(target_temperature != temperature)
 		var/difference = abs(target_temperature - temperature)
-		if(difference >= (10 * recharge_multiplier))
+		if(difference >= (recharge_multiplier))
```
Comment (Henri215) [inline code/modules/projectiles/guns/energy/special_eguns.dm:605]:
```suggestion
		if(difference >= recharge_multiplier)
```

File: code/modules/projectiles/guns/energy/special_eguns.dm [hunk 3]
```diff
@@ -597,17 +596,17 @@
...
 			if(target_temperature < temperature)
-				temperature -= (10 * recharge_multiplier)
+				temperature -= (recharge_multiplier)
```
Comment (Henri215) [inline code/modules/projectiles/guns/energy/special_eguns.dm:607]:
```suggestion
				temperature -= recharge_multiplier
```

File: code/modules/projectiles/guns/energy/special_eguns.dm [hunk 4]
```diff
@@ -597,17 +596,17 @@
...
 			else
-				temperature += (10 * recharge_multiplier)
+				temperature += (recharge_multiplier)
```
Comment (Henri215) [inline code/modules/projectiles/guns/energy/special_eguns.dm:609]:
```suggestion
				temperature += recharge_multiplier
```

File: code/modules/projectiles/guns/energy/special_eguns.dm [hunk 5]
```diff
@@ -545,16 +545,15 @@
...
 	// Measured in Kelvin
 	var/temperature = T20C
 	var/target_temperature = T20C
 	var/min_temp = 0
 	var/max_temp = 500
 
-	/// How fast the gun recharges
-	var/recharge_multiplier = 1
+	/// How fast the gun changes temperature
+	var/recharge_multiplier = 10
```
Comment (Contrabang) [inline code/modules/projectiles/guns/energy/special_eguns.dm:556]: Can you rename this var please, since the gun no longer self-charges?

**Review by DGamerL — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**